rules-threat-hunting/windows/process_creation/proc_creation_win_taskkill_execution.yml

```diff
@@ -20,16 +21,18 @@ detection:
     selection_img:
         - Image|endswith: '\taskkill.exe'
         - OriginalFileName: 'taskkill.exe'
-    selection_cli:
-        CommandLine|contains|all:
-            - ' /f'
+    selection_1_cli:
+        CommandLine|contains: ' /f'
+    selection_2_cli:
+        CommandLine|contains:
             - ' /im '
+            - ' /pid '
```

Comment:
Do want `/f AND (im OR pid)` or `/f OR im OR pid` ?
Ref doc https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/taskkill
there is a `/fi` for filter, so if we want only the forcefully  we need to write ` /f ` but the problem is that you just have to put the /f at the end to bypass the rule ...

Reply:
i want ```/f AND (im OR pid)```.

Reply:
So it is more `condition: all of selection_* and not 1 of filter_main_*`